resources/views/registerProject.blade.php

```diff
@@ -0,0 +1,218 @@
+					<label for="teamName">Team Name</label>
+					<input type="text" name="teamName" placeholder="Enter your team's name...">
+				</div>
+				<!-- Category -->
+					<!-- <div id="category" class="field">
+						<label>Category</label>
+						<div class="ui fluid search selection dropdown category">
+							<i class="dropdown icon"></i>
+							<input id ="category" type="hidden" name="country">
+							<div class="default text">Select a category of your project</div>
+							<div class="menu">
+								<div class="item" value="1">Category 1</div>
+								<div class="item" value="2">Category 2</div>
+								<div class="item" value="3">Category 3</div>
+								<div class="item" value="4">Category 4</div>
+							</div>
+						</div>
+					</div> -->
+				<div id ="category" class="field">
+					<label for="category">Category</label>
+					<select name="category" class="ui search dropdown" placeholder="Select your project's category">
+						<option style="color:grey;" value="0">Select your project's category</option>
+						<option value="1">Category 1</option>
+						<option value="2">Category 2</option>
+						<option value="3">Category 3</option>
+					</select>
+				</div>
+				<!-- Skillset -->
+				<div id="skillsets" class="field">
+					<label for="skillsets">Required Skillsets</label>
+					<div class="ui search">
+						<div class="ui icon input">
+							<input name="skillsets" class="prompt" type="text" placeholder="Search and add skillsets...">
+							<i class="search icon"></i>
+						</div>
+						<div class="results">
+						</div>
+					</div>
+				</div>
+				<!-- Milestones -->
+				<div id="milestones" class="field">
+					<label for="milestones">Public Milestones</label>
+					<input type="text" name="milestones" placeholder="e.g. Shipping">
+				</div>
+				<!-- Short Description -->
+				<div id="shortDescription" class="field">
+					<label>Brief Description</label>
+					<textarea rows="2"></textarea>
+				</div>
+				<!-- Long Description -->
+				<div id="longDescription" class="field">
+					<label>Detailed Description</label>
+					<textarea></textarea>
+				</div>
+				<!-- Video Pitch -->
+				<div id="videoPitch" class="field">
+					<label>Video pitch</label>
+					<input type="text" name="milestones" placeholder="Example: https://youtube.com/watch?v=238028302">
+				</div>
+				<!-- Register Button -->
+				<button id="registerProject" class="ui right floated button">Register project</button>
+			</div>
+		</div>
+	</div>
+</div>
+
+@section('jquery')
+<!-- Internal jQuery -->
+<script>
+	/* Search for Skillsets */
+	var skillsets = [
+		{ title: 'Skillset 1' },
+		{ title: 'Skillset 2' },
+		{ title: 'Skillset 3' },
+		{ title: 'Skillset 4' },
+		{ title: 'Skillset 5' },
+	];
+	$('.ui.search').search({source: skillsets});
+	$('.ui.search.category.dropdown').dropdown({useLabels: false});
+	$('.imgUploader').click(function(event) {
+		event.preventDefault();
+		$('#imgInput').click();
+	});
+
+	/*
```

Comment:
Lines 125-156 are all commented out. This shouldn't be the case. As previous: if you don't need them, please remove them.

Reply:
These lines are commented out because they aren't finished (They are commented on the top as changes still pending at the beginning of the conversation) I'll fix them up today.